io.github.TTimo.GtkRadiant.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
This file is not specific to Flathub; you should maintain it in your repo instead.